yky-social/src/yky.test.ts

```diff
@@ -9,19 +9,17 @@ import { Readable } from 'stream';
 
 import { describe, expect } from '@jest/globals';
 import request from 'supertest';
-import { YKY } from './app';
 
 import { PresignedPost } from '@aws-sdk/s3-presigned-post';
-import { Operations } from './YKYOperations';
-import { CurrentTimeCommunicator } from '@dbos-inc/communicator-datetime';
-import { BcryptCommunicator } from '@dbos-inc/communicator-bcrypt';
 
 import { TestingRuntime, createTestingRuntime } from '@dbos-inc/dbos-sdk';
 
+export { Operations } from './YKYOperations';
```

Comment:
Unfortunately the test runtime doesn't open dbos-config.yaml and read the entrypoints .js files, so I had to mention the name of one of the operations files from the test... otherwise nothing gets loaded.  Will put fixing on the TODO list for next week.

Reply:
Is it fixed by our latest release?

Reply:
I will check.  I did find an issue with the widget store also.  Time to dust off this PR (which somehow I forgot about).

Reply:
There was a long story about how I was not able to read the entrypoints from the .yaml file.
Because of when testRuntime is created in most tests, you have to set "NODE_OPTIONS=--experimental-vm-modules" which is... eww.  And once you've done that, if you create the test runtime more than once a different set of bad things happens.  So it was better to not do anything / more research is needed.